src/DonationForms/Listeners/StoreCustomFields.php

```diff
@@ -85,14 +86,23 @@ protected function storeAsDonationMeta(int $donationId, string $metaKey, $value)
     /**
      * @since 3.0.0
      */
-    protected function persistFieldScope(Field $field, $value, Donation $donation)
+    protected function storeAsSubscriptionMeta(int $subscriptionId, string $metaKey, $value)
+    {
+        give()->subscription_meta->update_meta($subscriptionId, $metaKey, $value);
+    }
```

Comment:
I'm not seeing this used anywhere. Do we want to add a scope for this?

Reply:
Added scope in here 2438b062